frontend/src/components/AppSearch.vue

```diff
@@ -34,9 +34,12 @@ export default {
       return this.$store.state.courses.map(item => item)
     },
     suggestions() {
+      // Give higher preference to course code before course name
       const lower = this.search.toLowerCase()
       return this.courses
-        .filter(item => item.tags.match(lower))
+        .filter(item => item.code.toLowerCase().match(lower))
```

Comment:
I think we should leave this as is since:
a) We want to display the codes as is.
b) If you are referring to changing the tags => might be useful in later search extensions?

Reply:
I agree with Papa - mainly cos fixing the data (the tags I guess) is outside the scope of this issue.
This is a simple fix which makes it behave more sensibly for now.

Reply:
Yeah I mean I don't really mind, it's just a backwards way to fix it. The above logic was right. It just needed the tags to be less liberal in the backend.